File: php-laraval-app/app/Models/ServiceQuarter.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class ServiceQuarter extends Model
{
    use HasFactory;

    /**
     * Indicates if the model should be timestamped.
     *
     * @var bool
     */
    public $timestamps = true;

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'service_id',
        'min_square_meters',
        'max_square_meters',
        'quarters',
    ];

    /**
     * The accessors to append to the model's array form.
     *
     * @var array
     */
    protected $appends = [
        'hours',
    ];

    protected $casts = [
        'created_at' => 'datetime',
        'updated_at' => 'datetime',
    ];

    /**
     * Define the columns from current model that is needed by the accessor.
     *
     * @var array<string,string[]>
     */
    protected array $accessorsFields = [
        'hours' => ['quarters'],
    ];

    public function getHoursAttribute(): float
    {
        return $this->quarters / 4;
    }

    public function service(): BelongsTo
    {
        return $this->belongsTo(Service::class)->withTrashed();
    }
}

File: dotnet/src/Downstairs.Infrastructure/Persistence/Models/ServiceQuarter.cs
namespace Downstairs.Infrastructure.Persistence.Models;

public partial class ServiceQuarter
{
    public long Id { get; set; }

    public long ServiceId { get; set; }

    public long MinSquareMeters { get; set; }

    public long MaxSquareMeters { get; set; }

    public long Quarters { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public virtual Service Service { get; set; } = null!;
}